### PR TITLE
Updating simulated player worker config

### PR DESCRIPTION
**Pull request by jessicafalk**
To make the simulated players work with https://github.com/spatialos/UnrealGDK/pull/1737

### CODE REVIEW

File: spatial/spatialos.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-  "name": "your_project_name_here",
+  "name": "unreal_gdk",
```
Comment (mironec) [inline spatial/spatialos.json]: I think this needs to be `your_project_name_here`

File: spatial/spatialos.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   "project_version": "0.0.1",
-  "sdk_version": "14.3.0",
+  "sdk_version": "14.4.0",
```
Comment (mironec) [inline spatial/spatialos.json]: We haven't updated the gdk's pinned version yet. I think we usually do all of it at once, but this probably doesn't change anything.

File: spatial/spatialos.json (hunk 1 above)
Comment (jessicafalk) [reply spatial/spatialos.json]: oh yeah, forgot to change that back

File: spatial/spatialos.json (hunk 2 above)
Comment (jessicafalk) [reply spatial/spatialos.json]: this is happening in this pr: https://github.com/spatialos/UnrealGDK/pull/1682

**Pull request merged**